### ISSUE TITLE
Use libmariadb-dev instead of libmariadbclient-dev

### ISSUE THREAD

**Issue by ottok** [ansible-community/ara-collection#8]
Hello!

Your current build uses libmariadbclient-dev. This is fine for now, but in the future when you update the Debian/Ubuntu version you are using in the CI, you should change those to libmariadb-dev (or libmariadb-dev-compat if you need to build something old that does not find the correct soname otherwise).

https://github.com/ansible-community/ara/search?q=libmariadbclient-dev

**Comment by Thulium-Drake** [comment]
The plot thickens, we don't really need this dependency at all :thinking: 

```
ara-ubt:~/playground# dpkg -l | grep mariadb 
ii  mariadb-client                 1:10.1.47-0ubuntu0.18.04.1          all          MariaDB database client (metapackage depending on the latest version)
ii  mariadb-client-10.1            1:10.1.47-0ubuntu0.18.04.1          amd64        MariaDB database client binaries
ii  mariadb-client-core-10.1       1:10.1.47-0ubuntu0.18.04.1          amd64        MariaDB database core client binaries
ii  mariadb-common                 1:10.1.47-0ubuntu0.18.04.1          all          MariaDB common metapackage
ii  mariadb-server                 1:10.1.47-0ubuntu0.18.04.1          all          MariaDB database server (metapackage depending on the latest version)
ii  mariadb-server-10.1            1:10.1.47-0ubuntu0.18.04.1          amd64        MariaDB database server binaries
ii  mariadb-server-core-10.1       1:10.1.47-0ubuntu0.18.04.1          amd64        MariaDB database core server files
```

These are all mariadb packages installed on my test ubuntu system. I ran the test task again also on my debian test system, which has no mariadb packages at all, still works fine.

I think it's something unrelated to mariadb, but it's somewhere in the ARA client. But I can't find where.

**Comment by ottok** [comment]
What about `dpkg -l | grep -e mariadb -e mysql`?

By the way, MariaDB 10.1 and Ubuntu 18.04 are getting pretty old by now. Recommend using e.g. MariaDB 10.3 and Ubuntu 20.04. There is even MariaDB 10.5 in newer distros.

**Comment by Thulium-Drake** [comment]
Agreed, but still doesn't matter :-) it doesn't work (I meant to post this on #15 ) so I don't think we need these libraries at all for ARA client systems.

My Debian test system has no mariadb/mysql packages whatsoever and it still functions with the ubuntu 18 system running ARA.
